contrib/libxls/utils/utils_libxls.cc

```diff
@@ -0,0 +1,222 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "contrib/libxls/utils/utils_libxls.h"
+
+#include <fstream>
+#include <iostream>
+#include <string>
+
+#include "contrib/libxls/sandboxed.h"
+
+absl::Status LibXls::OpenRemote() {
+  SAPI_RETURN_IF_ERROR(CheckIsInit());
+
+  sapi::v::CStr pcap_filename(filename_.c_str());
+  sapi::v::CStr pcap_encode(encode_.c_str());
+  sapi::v::IntBase<xls_error_t> pcap_error;
+
+  SAPI_ASSIGN_OR_RETURN(
+      xlsWorkBook * wb,
```

Comment:
NIT: Please left align pointer operator in declarations (multiple occurrences in the file)